README.md

```diff
@@ -1,81 +1,68 @@
-![small-white](https://user-images.githubusercontent.com/53123142/110364209-287bbd80-804c-11eb-91c3-a158b115ef27.png)
+<img src="https://user-images.githubusercontent.com/53123142/110364209-287bbd80-804c-11eb-91c3-a158b115ef27.png" width="650">
 
-roo.me is an open source web-application for easily finding roommates. 
 
-It works by showing users the best possible matches based on their profile information and preferences, with an easy 'yes or no' interface.
+# Description
+roo.me is an interactive and easy swipe web-app for finding roommate apartments.  
```

Comment:
Looks good!
maybe use something else instead of 'swipe' as we don't do that lol

Reply:
OK